src/modules/vtol_att_control/vtol_type.cpp

```diff
@@ -648,3 +648,32 @@ float VtolType::pusher_assist()
 	return forward_thrust;
 
 }
+
+float VtolType::getFrontTransitionTimeFactor() const
+{
+	// assumptions: transition_time = transition_true_airspeed / average_acceleration (thrust)
+	// transition_true_airspeed ~ sqrt(rho0 / rh0)
+	// average_acceleration ~ rho / rho0
+	// transition_time ~ sqrt(rho0/rh0) * rho0 / rho
+
+	// low value: hot day at 4000m AMSL with some margin
+	// high value: cold day at 0m AMSL with some margin
+	const float rho = math::constrain(_attc->getAirDensity(), 0.7f, 1.5f);
```

Comment:
maybe I'm being a bit over-scrupulous, but can we define these magic numbers as constants? since they have physical meaning (something with a comment about what standard altitude they come from so we can change it later if the application presents itself)  -- hoping to make this more common practice with new commits :)